CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - - Include credo checks
 - Update `EventBus.Model.Event` struct optional attribute type specs to allow `nil` values
 - Update license year
+- Change `EventBus.Service.Store.fetch` to return a safe value when ETS data is missing and log it
```

Comment:
If this is the expected change, it doesn't satisfy the implementation.

Reply:
Going to add the logging back again.